### ISSUE TITLE
The SM gibs you even if you have the funny shoes

### ISSUE THREAD

**Issue by trerri** [tgstation/tgstation#51794]
Issue reported from  Round ID: [140184](https://scrubby.melonmesa.com/round/140184) (/tg/Station Terry [EU] [100% LAG FREE] [New-Dedi])

Reporting client version: 513.1526

The new completely unnecessary, unwanted, and nonsensical "SM pulling you in based on EER" feature just dropped. It ignores magboots. The SM chamber has 0 tiles between you and the instagib crystal. Now reliably editing the SM on any half-decent setup is completely impossible, even with the previously established "wear funny boot and do not click middle square directly" method.
If this feature got the OK because it negates the "100 billion items next to the SM for beeg power", know that it's a bad, band-aid fix.
The real fix for that is removing radioactive contamination altogether: it's unrealistic, unfun, and lags very much. There is no way to remove it in large amounts without AFKing at a shower or basically killing yourself in a suit unit (and that usually doesn't even clear it). Since there is no downside to removing it if you have 100% rad protection, you are actively encouraged to poison everyone around you. Radiation is already a chore on servers where you can see exactly how long it lasts and reduce it without taking off the suit, on /tg/ it's just the worst thing.

TL;DR: The easy fix is to make it not ignore magboots. The good fix is to revert this and also remove radioactive contamination.
